build/package/Microsoft.DotNet.Cli.Installer.DEB.targets

```diff
@@ -86,14 +93,8 @@
       <DebianConfigTokenValues Include="%SHARED_FRAMEWORK_DEBIAN_PACKAGE_NAME%">
         <ReplacementString>$(SharedFxDebianPackageName)</ReplacementString>
       </DebianConfigTokenValues>
-      <DebianConfigTokenValues Include="%SHARED_FRAMEWORK_NUGET_NAME%">
```

Comment:
also, why you're looking at this, is CLI repo still the producer of the real Core-Setup DEB packages or can we remove some of this code?